redisdb/check.py

```diff
@@ -226,6 +226,8 @@ def _check_db(self, instance, custom_tags=None):
         # Save the number of commands.
         self.rate('redis.net.commands', info['total_commands_processed'],
                   tags=tags)
+        self.gauge('redis.net.commands.instantaneous', info['instantaneous_ops_per_sec'],
```

Comment:
why not use the full name `instantaneous_ops_per_sec` ? I don't think 'instantaneous` is clear enough alone and it's the name use the in the INFO command/doc.